src/pickup.h

```diff
@@ -0,0 +1,37 @@
+#ifndef _PICKUP_H
+#define _PICKUP_H
+
+#include <unordered_map>
+
+#include "entity.h"
+
+namespace pickupNS {
+	const enum PICKUP_TYPE { MAGNET, INVERT };
```

Comment:
I can't load enums dynamically from a `json` file 😢. There is a map converting `strings` in the json file to its respective enum value. It is declared in `pickupManager.cpp` 

The json file is still okay as it stores the frame and duration of each effect.